committee-code-of-conduct/governance/incident-process.md

```diff
@@ -0,0 +1,111 @@
+# Incident reporting and response process
+
+This document outlines the Code of Conduct Committee's workflow when receiving and responding to an incident report. As each report is unique, the process is described at a high level.
+
+## When and Where does the Kubernetes Code of Conduct apply?
+
+### What is an incident report?
+
+An **incident report** is a description of an event, interaction, or public statement submitted to the Kubernetes Code of Conduct Committee, which the reporter feels violates the [Kubernetes Code of Conduct](https://kubernetes.io/community/code-of-conduct/). 
+
+### Who can submit a report? 
+
+The Code of Conduct Committee accepts reports from everyone who interacts with the Kubernetes project community, contributor or otherwise. This includes, but is not limited to, the following:
+
+- Contributors and maintainers
+- Members of the Kubernetes Slack instance 
+- Attendees and vendors at KubeCon/CloudNativeCon
+- CNCF Ambassadors
+- Vendors/companies/projects which use Kubernetes and need to interact with the community as a result
+
+At times we encourage community members to email us if an incident is ongoing and we have not been contacted.
+
+### What are the boundaries of the Kubernetes community?
+
+There are no hard boundaries of the community, but common places we are asked to extend guidance to are:
+
+- Official Kubernetes communication channels
+- Kubernetes events
+- Media and web presences
+- Social media
+    - In some cases, where individual social media messages are not related to Kubernetes but have been reported to the Code of Conduct Committee and are making project members feel unsafe or unwelcome, we might choose to act.
+
+### Where do private incident reports happen? 
+
+The Code of Conduct Committee's primary means of contact is our email address, conduct@kubernetes.io. 
+
+We can also be reached via Slack direct messages to individual committee members (see [member list](https://github.com/kubernetes/community/tree/master/committee-code-of-conduct#members)) or otherwise, though we might direct you to contact us via email. 
+
+### How is the privacy of a report protected?
+
+### Why does this process exist?
+
+The reporting process exists to provide the community with mechanisms to keep people safe, and to ensure that poor behavior, regardless of who the initator is, is not accepted.
+
+The Code of Conduct Committee has unilateral power to address harms as needed and appropriate to restore community safety after any incident(s). We are seperate from the Steering Committee and all other bodies in the Kubernetes community to provide a mechanism by which anyone can report, regardless of regardless of roles and organizational power dynamics which often lead to systemic underreporting.
```

Comment:
+1

Reply:
typo: "regardless  of" is repeated twice